Functions.Templates/Templates/HttpTrigger-Python-Preview/function_app.py

```diff
@@ -0,0 +1,27 @@
+import azure.functions as func
+
+import logging
+
+app = func.FunctionApp(auth_level=func.AuthLevel.ANONYMOUS)
+
+@app.function_name(name="HttpTrigger1")
```

Comment:
As this template is intended to be appended to an existing `function_app.py`, I would think it should not include imports or the creation of the `app`.

Reply:
agreed, have removed